Src/Plugins/Tests.Security/Validators/SEC101_015.AkamaiCredentialsValidatorTests.cs

```diff
@@ -18,7 +18,7 @@
 namespace Microsoft.CodeAnalysis.Sarif.PatternMatcher.Plugins.Security.Validators
 {
     /// <summary>
-    /// Testing SEC101/026.AkamaiCredentialsValidatorTests
+    /// Testing SEC101/015.AkamaiCredentialsValidator
```

Comment:
>AkamaiCredentialsValidator [](http://example.com/codeflow?start=27&length=26)

Well done fixing all these comments up, Ed!!

Reply:
thanks!
i'm following ur guidance to make everything in the same format, it will facilitate in the future :)